analytics-webapps/src/main/webapp/vue-app/generic-portlet/settings/AnalyticsChartSetting.vue

```diff
@@ -15,20 +15,20 @@
           @click="dialog = false">
         </a>
         <span class="ignore-vuetify-classes PopupTitle popupTitle">
-          Edit chart settings
+          {{ $t('analytics.label.edit.chart.settings') }}
         </span>
       </div>
       <v-card-text>
         <v-tabs
           v-model="tab"
           background-color="transparent"
           color="primary">
-          <v-tab>General</v-tab>
-          <v-tab>Colors</v-tab>
-          <v-tab>X axis</v-tab>
-          <v-tab>Y axis</v-tab>
-          <v-tab>Multiple charts</v-tab>
-          <v-tab>Data filters</v-tab>
+          <v-tab>{{ $t('analytics.settings.tab.General') }}</v-tab>
```

Comment:
It's better to always use simple keys for labels to be able to reuse them:
analytics.settings.tab.General => analytics.general
analytics.settings.tab.Colors => analytics.colors

Imagine you add in another place "Colors" label. Is it better to define a new key or reuse the existing key ?
In fact, it's better to reuse it to ensure that we don't increase the number of keys to translate for nothing.

(Same for other keys)

Reply:
OK I will do it